internal/database/repos.go

```diff
@@ -706,6 +706,10 @@ type ReposListOptions struct {
 	// last_error value in the gitserver_repos table.
 	FailedFetch bool
 
+	// IsCorrupted, if true, will filter to only repos that where corruption has been detected.
+	// A repository is corrupt in the gitserver_repos table it has a non-null value for corruptedAt
+	IsCorrupted bool
```

Comment:
```suggestion
	OnlyCorrupted bool
```

Nitpick: to stay consistent with the other options, we should use the same naming scheme (even though I'm not convinced the `Only*` scheme is the greatest thing we could've come up with)